modules/ipi-install-configuring-host-network-interfaces-in-the-install-config.yaml-file.adoc

```diff
@@ -2,62 +2,110 @@
 //
 // installing_bare_metal_ipi/ipi-install-installation-workflow.adoc
 
+:_content-type: PROCEDURE
 [id="configuring-host-network-interfaces-in-the-install-config.yaml-file_{context}"]
 = (Optional) Configuring host network interfaces in the `install-config.yaml` file
 
-During installation, you can set the `networkConfig` configuration setting in the `install-config.yaml` file to configure host network interfaces using NMState. To use the `networkConfig` configuration setting, you must provide an NMState YAML configuration.
+During installation, you can set the `networkConfig` configuration setting in the `install-config.yaml` file to configure host network interfaces using NMState.
 
 [IMPORTANT]
 ====
-Before configuring host network interfaces, review the OpenShift Container Platform 4.10 release notes.
+Host network interface configuration is set during the installation and provisioning process and includes advanced options, such as setting static IPs per host. Before configuring host network interfaces, review the OpenShift Container Platform 4.10 release notes. 
 ====
 
- See link:https://nmstate.io/examples.html#interfaces-ethernet[NMState] for additional examples of the NMState syntax.
+.Prerequisites
 
+. For each node being configured with a static IP, you must configure a DNS PTR record containing a valid host name including the specific IP associated with that node.
+
+.Procedure
+
+. (Optional) Consider saving the `networkConfig` YAML syntax to a file.
+
+.. Test it using the NMState command line interface. Do this before including it in the `install-config.yaml` file, because the installation program will not check the NMState YAML syntax. 
+
+.. Execute `nmstatectl gc <yaml-config>` to test the syntax. Errors in the YAML syntax might result in a failure to apply the network configuration. 
+
+.. Maintain the valid YAML syntax when applying changes using Kubernetes NMState after deployment or when expanding the cluster.
+
+. To use the `networkConfig` configuration setting, you must provide an NMState YAML configuration. In the following example, the user name is `user` and the password is `password`:
```

Comment:
What must you do to provide the config? "Create and save an NMState configuration as a YAML file, as shown in the following example"?